governance-exercises/VoteWizard_poll_exercise_01/Adjust Spork D3M Parameters and DAI Interest Rate Strategy - July 24, 2023.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Adjust Spork D3M Parameters and DAI Interest Rate Strategy - July 24, 2023
```

Comment:
The clue on atomization can come from titles a fair bit. If you find yourself writing a poll where the title has an 'and' in it, give serious consideration to dividing it up more.